### ISSUE TITLE
Use ${intensity.percent} with Harmony Hub

### ISSUE THREAD

**Issue by raleph** [bwssytems/ha-bridge#416]
It would be great to be able to use ${intensity.percent} with Harmony Hub buttons. 
I have buttons like "Ventilator 1", "Ventilator 2", etc. in the hub to kind of "dim" my fan.
When I set up a ON command in HA-bridge like

`[{"device":"123456","button":"Ventilator 1"}]`

it works fine. 
But when I set up a dim command using 

`[{"device":"123456","button":"Ventilator ${intensity.percent}"}]`

I get error `Failed communicating with Harmony Hub` in the logs. It looks like ${intensity.percent} does not get replaced by the respective value.
Am I doing something wrong here or is this just not implemented (yet)?

**Comment by bwssytems** [comment]
Currently, the ${intensity variables are not used in the harmony items as they seemed to be pretty static. Yet you found a way to use that! The use case though is when you say 'Alexa, set ventilator to 2' and that would pick the correct one. Is this what you are after?

**Comment by raleph** [comment]
By saying 'Alexa, set ventilator to 2' I intend to initiate button "Ventilator 2" thus choosing intensity level 2 of of my fan.

**Comment by bwssytems** [comment]
But, this doesn't seem much different than  "Alexa, turn on ventilator 2"

**Comment by raleph** [comment]
Sorry, there might be a misunderstanding. I have just one fan. 
The fan can be set to three levels of intensity by pressing button 1, 2 or 3 on it's remote. I have set up the three buttons in Harmony and would like to call them via the ${intensity.percent} variable as described above. 
The same approach could be used to switch channels 1-9 on a tv for example.

**Comment by bwssytems** [comment]
Ok, I understand now. But, you could have the name of the device as `Ventilator Level Two` and that would be `Alexa, Turn on Ventilator Level 2`

**Comment by aetolus** [comment]
If at all possible, I'd like to second this as a feature request - was actually just going to make my own thread before I glanced through the history.

My use case is similar - I have a Fujitsu split system in my Harmony Hub that doesn't return state requests. So I've created buttons spanning 18 - 30 representing degrees Celsius. 

It'd be great to have this functionality as a dim setting so I don't have to build a button the Echo can see, but rather can just select the temperature. 

I've done similar using the emulated_hue component in HASS which I'm hoping to replace with ha-bridge.
